### DISCUSSION TITLE
Iterate molecule/rating component

### DISCUSSION THREAD

**Discussion by nistadev** [SUI-Components/sui-components#1626] [Ideas and new features]
Due to a new development, we found that `molecule/rating` component is too _simple_, and we will like to add some improvements. Here are some bullet-points to give context:

- We had to develop a new form where users need to evaluate stuff, so our _inputs_ are ratings.
- This component has an open issue (#1385) which improves performance and is a **major** release but does not solve our problem.
- This is how we would like it to improve: add some status, different icons for hover/selected/default, add labels to stars & legend instead of just one label, choose star color with tokens. 

![image](https://user-images.githubusercontent.com/22194171/126995123-4f2f9914-6bfa-4b00-ab06-f61a512a74ed.png)

**As this are breaking changes**, we would like to share the idea with the community and discuss if there are others who need this states & features too. We are open to collaborate, iterate and develop this component asap.

**Comment by turolopezsanabria** [comment]
Should we consider `hover` for filled stars?
